src/components/DefaultPageLayout.tsx

```diff
@@ -62,7 +62,10 @@ const DefaultPageLayout = ({
   const toast = useToast();
 
   useEffect(() => {
-    document.title = intl.formatMessage({ id: titleId });
+    const appName = `micro:bit ${TOOL_NAME}`;
+    document.title = titleId
+      ? `${intl.formatMessage({ id: titleId })} | ${appName}`
+      : appName;
```

Comment:
We somehow lost the logic where pages in the tool have titles "${name of page} | micro:bit AI creator" e.g. "Data samples | micro:bit AI creator".